api-admin/src/main/java/bio/terra/pearl/api/admin/service/enrollee/EnrolleeImportExtService.java

```diff
@@ -0,0 +1,77 @@
+package bio.terra.pearl.api.admin.service.enrollee;
+
+import bio.terra.pearl.api.admin.service.AuthUtilService;
+import bio.terra.pearl.core.model.EnvironmentName;
+import bio.terra.pearl.core.model.admin.AdminUser;
+import bio.terra.pearl.core.model.dataimport.Import;
+import bio.terra.pearl.core.model.study.PortalStudy;
+import bio.terra.pearl.core.model.study.StudyEnvironment;
+import bio.terra.pearl.core.service.CascadeProperty;
+import bio.terra.pearl.core.service.dataimport.ImportService;
+import bio.terra.pearl.core.service.exception.NotFoundException;
+import bio.terra.pearl.core.service.exception.PermissionDeniedException;
+import bio.terra.pearl.core.service.export.EnrolleeImportService;
+import bio.terra.pearl.core.service.study.StudyEnvironmentService;
+import java.io.InputStream;
+import java.util.List;
+import java.util.UUID;
+import org.springframework.stereotype.Service;
+
+@Service
+public class EnrolleeImportExtService {
+  private EnrolleeImportService enrolleImportService;
+  private StudyEnvironmentService studyEnvironmentService;
+  private AuthUtilService authUtilService;
+  private ImportService importService;
+
+  public void EnrolleImportExtService(
+      EnrolleeImportService enrolleImportService,
+      AuthUtilService authUtilService,
+      ImportService importService,
+      StudyEnvironmentService studyEnvironmentService) {
+    this.enrolleImportService = enrolleImportService;
+    this.authUtilService = authUtilService;
+    this.importService = importService;
+    this.studyEnvironmentService = studyEnvironmentService;
+  }
+
+  public List<Import> list(String portalShortcode, AdminUser operator) {
+    authUtilService.authUserToPortal(operator, portalShortcode);
+    return importService.findAll();
+  }
+
+  public Import importData(
+      String portalShortcode,
+      String studyShortcode,
+      EnvironmentName environmentName,
+      InputStream tsvData,
+      AdminUser operator) {
+    authUtilService.authUserToPortal(operator, portalShortcode);
+    PortalStudy portalStudy =
+        authUtilService.authUserToStudy(operator, portalShortcode, studyShortcode);
+    StudyEnvironment studyEnv =
+        studyEnvironmentService.verifyStudy(studyShortcode, environmentName);
+    return enrolleImportService.importEnrollees(
+        portalShortcode, studyShortcode, studyEnv, tsvData, operator.getId());
+  }
+
+  public void delete(
```

Comment:
mostly for my own curiosity again, but IIRC from the design discussion of the import system, this delete would also delete the enrollees created so that you could import the enrollees and make sure everything worked before finalizing it. is that accurate to how this should work? I notice it's deleting the import audit trail, but I wasn't sure if it should be doing more.

Reply:
Honestly I added it incase we need it. From front-end we might not need deleting full import, only import items. 
Will need more discussion and might remove it during front-end integration.
@devonbush 
We dont want to delete full import, right ? 
Should I remove import delete end point or leave it. 
Maybe I should add importItem delete endpoint

Reply:
will leave it for now and change it if needed.

Reply:
Yeah, I've been waffling on that -- to the extent that imports and importItems are an audit trail of imports, we might not want to delete them ever.  That way an admin can see their history of "yesterday you did an import -- it imported 60 participants but five of them failed, then you deleted those participants and retried, and only 2 failed".  So maybe  instead of a hard delete, we'll want an additional status on import and importItem of "deleted"

Reply:
will take care of it my next PR, Devon

Reply:
would be good use of importStatus